src/inspector/models/general/generalsettingsmodel.cpp

```diff
@@ -33,8 +33,12 @@ GeneralSettingsModel::GeneralSettingsModel(QObject* parent, IElementRepositorySe
 
     setTitle(qtrc("inspector", "General"));
     setSectionType(InspectorSectionType::SECTION_GENERAL);
-    setPlaybackProxyModel(new PlaybackProxyModel(this, repository));
-    setAppearanceSettingsModel(new AppearanceSettingsModel(this, repository));
+
+    m_playbackProxyModel = new PlaybackProxyModel(this, repository);
+    m_playbackProxyModel->init();
+
+    m_appearanceSettingsModel = new AppearanceSettingsModel(this, repository);
+    m_appearanceSettingsModel->init();
```

Comment:
You should make a similar change for BeamSettingsModel

Reply:
RestBeamSettingsModel

Reply:
@cbjeukendrup please fix this for the master https://github.com/musescore/MuseScore/pull/18531/commits/0423c039fddada315f0f6a79e8bcf9004fbd9d15

Reply:
Finally done, sorry for the inconvenience